atomic

```diff
@@ -277,6 +277,13 @@ if __name__ == '__main__':
         action="store_true",
         help=_("preview the command that %s would execute") % sys.argv[0])
     installp.add_argument("image", help=_("container image"))
+    installp.add_argument("--system", dest="system",
+                          action='store_true', default=False,
+                          help=_('install a system container'))
+    installp.add_argument("--set", dest="setvalues",
+                          action='append',
+                          help=_("Specify a variable in the VARIABLE=VALUE "
+                                 "form for a system container"))
     installp.add_argument("args", nargs=argparse.REMAINDER,
```

Comment:
If I call --set on a non --system container are we checking for this and exiting out?